app/extensions.js

```diff
@@ -197,6 +198,13 @@ module.exports.init = () => {
   let enableExtensions = () => {
     installExtension(config.braveExtensionId, getExtensionsPath('brave'), {manifest_location: 'component', manifest: generateBraveManifest()})
 
+    if (getSetting(settings.PDFJS_ENABLED)) {
```

Comment:
In general for "rich" (as in pwnable) legacy media types handled by extensions, we want a setting to disable.